src/bundles/stereo_sound/functions.ts

```diff
@@ -450,26 +451,13 @@ export function play(sound: Sound): AudioPlayed {
     riffwave.header.bitsPerSample = 16;
     riffwave.Make(channel);
 
-    /*

```

Comment:
This part is still found it `sound/functions.ts` in L395, is it still needed there?